conandata.yml

```diff
@@ -3,10 +3,19 @@ version: "5.10.0-alpha.0"
 requirements:
   - "pyarcus/5.10.0@ultimaker/stable"
 
+extra_dependencies:
+  Qt:
+    version: "6.6.0"
+    sources_url: https://code.qt.io/cgit/qt/qtbase.git
+    home_url: https://github.com/Ultimaker/Cura/blob/main/licenses_thirdparty/qt.md
```

Comment:
I'm not sure you changed the right one here? Do we still want to link to the original qtbase repo in the `sources_url`?